src/main/kotlin/uk/gov/dwp/dataworks/kafkatohbase/reconciliation/repositories/impl/HBaseRepositoryImpl.kt

```diff
@@ -70,10 +82,13 @@ class HBaseRepositoryImpl(private val connection: Connection, private val tableN
 
     private var _mutableMap: MutableMap<String, Boolean> = mutableMapOf()
 
-    private fun get(id: String, version: Long) = Get(tableNameUtil.decodePrintable(id)).apply {
+    private fun get(id: String, version: Long, replicaId: Int) = Get(tableNameUtil.decodePrintable(id)).apply {
         setTimeStamp(version)
         isCheckExistenceOnly = true
         consistency = Consistency.TIMELINE
+        if (replicaId > 0) {
```

Comment:
Needs to be >=

Reply:
I would add a unit test specifically checking passing in 0 and using 0 to ensure that is caught.